packages/manager/modules/bm-server-components/src/netboot/constants.js

```diff
@@ -0,0 +1,34 @@
+export const NETBOOT_GUIDES = {
+  de_DE:
+    'https://docs.ovh.com/de/dedicated/diagnose-hardwarefehler-dedicated-server',
+  en_ASIA: 'https://docs.ovh.com/asia/en/dedicated/hardware-diagnostics',
+  en_AU: 'https://docs.ovh.com/au/en/dedicated/hardware-diagnostics',
+  en_CA: 'https://docs.ovh.com/ca/en/dedicated/hardware-diagnostics',
+  en_GB: 'https://docs.ovh.com/gb/en/dedicated/hardware-diagnostics',
+  en_IE: 'https://docs.ovh.com/ie/en/dedicated/hardware-diagnostics',
+  en_SG: 'https://docs.ovh.com/sg/en/dedicated/hardware-diagnostics',
+  en_US: 'https://docs.ovh.com/us/en/dedicated/hardware-diagnostics',
+  es_ES:
+    'https://docs.ovh.com/es/dedicated/diagnostico-fallos-hardware-servidor-dedicado',
+  es_US:
+    'https://docs.ovh.com/us/es/dedicated/diagnostico-fallos-hardware-servidor-dedicado',
+  fr_CA:
+    'https://docs.ovh.com/ca/fr/dedicated/diagnostic-dysfonctionnements-materiels-serveur-dedie',
+  fr_FR:
+    'https://docs.ovh.com/fr/dedicated/diagnostic-dysfonctionnements-materiels-serveur-dedie',
+  it_IT:
+    'https://docs.ovh.com/it/dedicated/diagnostica-problemi-hardware-server-dedicato',
+  pl_PL:
+    'https://docs.ovh.com/pl/dedicated/diagnostyka-usterek-sprzetowych-serwera-dedykowanego',
+  pt_PT:
+    'https://docs.ovh.com/pt/dedicated/diagnostico-avarias-materiais-servidor-dedicado',
```

Comment:
are you sure you have to consider language ?
from Manager POV, links are subsidiary related not language.
I guess weird to have some part subsidiary related and others language.

I guess a global handling way for all manager is best.

Reply:
Sample based on subsidiary:

```suggestion
  FR:
    'https://docs.ovh.com/fr/dedicated/diagnostic-dysfonctionnements-materiels-serveur-dedie',
  IT:
    'https://docs.ovh.com/it/dedicated/diagnostica-problemi-hardware-server-dedicato',
  PL:
    'https://docs.ovh.com/pl/dedicated/diagnostyka-usterek-sprzetowych-serwera-dedykowanego',
  PT:
    'https://docs.ovh.com/pt/dedicated/diagnostico-avarias-materiais-servidor-dedicado',
```

Reply:
@mohammed-zahaf do you have an example elsewhere in the manager-ovh code where documentation links have been implemented without any language consideration ?

Reply:
yes you can consider this:
packages/manager/modules/pci/src/projects/project/storages/volume-backup/volume-backup.constants.js#116

Reply:
to get the subsidiary you can use the coreConfig module.
`coreConfig.getUser().ovhSubsidiary`

Reply:
Does it really depends on the language that has been set in the manager-ovh ?
![image](https://github.com/ovh/manager/assets/17429937/a1c3e311-50d1-4e79-814f-99f1b653b239)
If I remember correctly, it depends on the APIv6 /me call subsidary attribute that is generated once at the nic creation
![image](https://github.com/ovh/manager/assets/17429937/24470f0b-452d-40b9-be94-3e318b3289e1)

Reply:
No, currently it's not depend on language but on subsidiary.
I understand you need that the link depend on language not on subsidiary.

My point is, if you do so we will have some part depend on language and some others parts depend on subsidiary.
Having both it will be frustrating for customers experience. That why i suggest to use same pattern as Manager implement -> subsidiary not language.

I let you discuss with your PM,PO.